README.md

```diff
@@ -1 +1,128 @@
-# `setup-gazebo`
\ No newline at end of file
+# `setup-gazebo`
+
+This action sets up a Gazebo release inside a Linux environment.
+
+1. [Overview](#Overview)
+1. [Supported Gazebo releases](#Supported-Gazebo-releases)
+1. [Tasks performed by the action](#Tasks-performed-by-the-action)
+1. [Usage](#Usage)
+    1. [Setting up worker and installing a compatible Gazebo and Ubuntu combination](#Setting-up-worker-and-installing-a-compatible-Gazebo-and-Ubuntu-combination)
+    1. [Iterating on all Gazebo and Ubuntu combinations](#Iterating-on-all-gazebo-ubuntu-combinations)
+1. [License](#License)
+
+## Overview
+
+The `setup-gazebo` GitHub Action sets up an environment to install a Gazebo release in a compatible Ubuntu distribution. The action requires a Gazebo release name as an input for the `required-gazebo-distributions` field.
+
+It is recommended to use the `setup-gazebo` action inside a Docker container.
+
+## Supported Gazebo releases
+
+This action currently supports the installation of four Gazebo [releases]. The following table lists the release name, EOL date and the supported operating system.
+
+| Release Name | EOL Date | Supported OS |
```

Comment:
Note that the goal is not to only support that Ubuntu platforms. Harmonic is supported in Noble and even the code that we have in the repository something like:

```yaml
  jobs:
    build_docker:
      runs-on: ubuntu-latest
      container:
        image: ubuntu:noble
      steps:
        - name: 'Setup Gazebo'
          uses: gazebo-tooling/setup-gazebo@<full_commit_hash>
          with:
            required-gazebo-distributions: harmonic
        -run: 'gz sim --versions'
```

The table is useful but I'm afraid of having to duplicate the information of https://gazebosim.org/docs/harmonic. Not sure if it would be better to remove it completely and point the people to the docs or to have it here with full documentation on supported platforms.